oneargopy/Argo.py

```diff
@@ -507,12 +626,28 @@ def __mark_bgcs_in_prof(self):
         is_bgc = self.prof_index['wmoid'].isin(bgc_floats)
         self.prof_index.insert(1, "is_bgc", is_bgc)
 
-    def __load_is_bgc_index(self)-> pd:
-        """ Function to create a dataframe with float IDs and
-            their is_bgc status for use in select_profiles().
+
+    def __load_float_stats(self)-> pd:
+        """ Function to create a dataframe with float IDs,
+            their is_bgc status, and their most recent update
+            date for use in select_profiles().
```

Comment:
Data for physical floats are taken from the prof index file, which data for BGC floats are taken from the Sprof index file.